packages/twenty-server/src/database/commands/upgrade-version/0-32/0-32-copy-webhook-operation-into-operations-command.ts

```diff
@@ -43,17 +43,26 @@ export class CopyWebhookOperationIntoOperationsCommand extends ActiveWorkspacesC
 
       for (const webhook of webhooks) {
         if ('operation' in webhook) {
-          let newOperation = webhook.operation;
+          let newOpe = webhook.operation;
+
+          newOpe = newOpe.replace(/\bcreate\b(?=\.|$)/g, 'created');
```

Comment:
bit surprise about \b operator. shouldn't it be ^ (start with) direclty?
Not a problem just curious about this \b operator which is not super usual in regex

Reply:
yes, i will update that, I just used the chatgpt -> 101 validated REGEX

Reply:
\b is checking word boudaries -> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Word_boundary_assertion
<img width="416" alt="image" src="https://github.com/user-attachments/assets/27a641c1-854a-4121-84c6-819146b97e27">

Reply:
might work with ^but it has sense